plugins/dev-tools/src/block_test_helpers.mocha.js

```diff
@@ -155,13 +176,21 @@ export const runSerializationTestSuite = (testCases) => {
    */
   const createXmlRoundTripTestCallback = (testCase) => {
```

Comment:
Could we update the name of this and the above? Either by removing `Xml` or by adding `createXmlOrJson...`?

Reply:
We can, but it would be a breaking change :/ Just confirming that's ok before I change it?